requirements/base.txt

```diff
@@ -1,10 +1,9 @@
-mock==2.0.0
-responses==0.10.15
-google-resumable-media==1.1.0
-googleapis-common-protos==1.6.0
-google-cloud-storage==1.32.0
-better-setuptools-git-version==1.0.5
-unittest-xml-reporting==3.0.2
-wheel==0.35.1
-setuptools==49.6.0
-jinja2<2.11.0,>=2.10.1
\ No newline at end of file
+google-cloud-storage>=1.32,<2
+google-auth>=1.20,<2
+
+better-setuptools-git-version>=1.0.5,<2
+unittest-xml-reporting>=3.0.2,<4
+jinja2>=2.10,<3
+
+wheel>=0.35
+setuptools>=49
```

Comment:
>=49,<50?

Reply:
AFAIK pinning 'setuptools' might break some 'venv' related tools (but can't find clear details about this).
Also 'setuptools' are used to install package. If version of installed setuptools differ from version in 'bigflow' - package might be reinstalled during it's usage.

Reply:
BTW: 'pip-tools' doesn't include version of 'setuptools' into generated 'requirements.txt' file on purpose.
https://github.com/jazzband/pip-tools/blob/61eb515b430a57e71f0bf768049e6a1f23d3c7c8/piptools/utils.py#L17

Reply:
ok, closing this comment